scylla-cdc-replicator/src/replicator_consumer.rs

```diff
@@ -13,20 +13,30 @@ use scylla::Session;
 
 use scylla_cdc::consumer::*;
 
-pub(crate) struct ReplicatorConsumer {
```

Comment:
The diff of the first commit is very hard to read and review... This looks like a refactor of multiple things at once. Could you break it into multiple commits, each commit representing a single refactoring step so that the commit diffs are easier to read?

Reply:
Actually, a little change in `ReplicatorConsumer` struct causes changes in many places, so a single-refactoring-step commit will also be big if want to make sure that each commit compiles and passes all tests. I will try to break it down into smaller ones.

Reply:
It's OK if steps are large, as long as changes they introduce are easy to understand. For example, a step can be:

- Introducing a sub-struct and moving some fields there. If the fields are used in many places, then the diff will be large but most changes will be trivial (changing `self.field` to `self.substruct.field`).
- Converting a method into a stand-alone function,
- Changing a method to precompute prepared queries instead of always sending unprepared queries.